test/cypress/integration/widgets/deployment_num_spec.ts

```diff
@@ -55,7 +55,7 @@ describe('Number of Deployments widget', () => {
         function setWidgetConfiguration(filterId: string, pageToOpenOnClick: string) {
             cy.editWidgetConfiguration(widgetId, () => {
                 cy.setSearchableDropdownValue('Filter ID', filterId);
-                cy.setDropdownValues('Page to open on click', [pageToOpenOnClick]);
+                cy.setMultipleDropdownValues('Page to open on click', [pageToOpenOnClick]);
```

Comment:
It shouldn't be `setMultipleDropdownValues`. 

It uses the same page selector as "Number of Blueprints" widget, so we should change it to:
```suggestion
                cy.setSingleDropdownValue('Page to open on click', pageToOpenOnClick);
```